src/EventStore.ClientAPI/Projections/ProjectionsClient.cs

```diff
@@ -142,9 +142,10 @@ public Task<string> GetQuery(EndPoint endPoint, string name, UserCredentials use
             return SendGet(endPoint.ToHttpUrl(httpSchema, "/projection/{0}/query", name), userCredentials, HttpStatusCode.OK);
         }
 
-        public Task UpdateQuery(EndPoint endPoint, string name, string query, UserCredentials userCredentials = null, string httpSchema = EndpointExtensions.HTTP_SCHEMA)
+        public Task UpdateQuery(EndPoint endPoint, string name, string query, UserCredentials userCredentials = null, string httpSchema = EndpointExtensions.HTTP_SCHEMA, bool emitEnabled = true)
```

Comment:
Defaulting the emit enabled to true could have unintended consequences.
The expected behaviour is if it's not set, the value will not be changed.
However, in this case, there is no way to _not_ set emit enabled.

Perhaps it would be better to create an override method that accepts the emitEnabled parameter, and only if that override is used will it be sent as part of the projection update.

Reply:
OK, so you wanna make this method accepts bool?, and if not set it will be execlude emit paramater from Url, is that right ?

Reply:
Sounds good 👍

Reply:
@hayley-jean applied PR comments. please have a look. Thanks.